client/my-sites/domains/domain-management/list/index.jsx

```diff
@@ -110,24 +108,19 @@ export class List extends React.Component {
 		const { translate } = this.props;
 
 		return (
-			<Notice
-				status="is-success"
-				showDismiss={ false }
-				text={ translate( 'Free domain available' ) }
-				icon="info-outline"
+			<UpsellNudge
+				callToAction={ translate( 'Claim Free Domain' ) }
```

Comment:
Are these button texts supposed to be sentence-case?

Reply:
Ayuh :)